docker/compose/docker-compose-base.yml

```diff
@@ -1,66 +0,0 @@
-postgres:
-  image: dimagi/docker-postgresql
-  environment:
-    POSTGRES_USER: commcarehq
-    POSTGRES_PASSWORD: commcarehq
-  labels:
-    commcare.name: "postgres"
-  expose:
-    - "5432"
-
-couch:
-  image: klaemo/couchdb:latest
-  labels:
-    commcare.name: "couch"
-  expose:
-    - "5984"
-
-redis:
-  image: redis
-  labels:
-    commcare.name: "redis"
-  expose:
-    - "6379"
-
-elasticsearch:
-  build: ../dockerfiles
-  dockerfile: Dockerfile_es
-  environment:
-    ES_JAVA_OPTS: "-Des.script.engine.groovy.inline.aggs=true -Des.script.engine.groovy.inline.search=true"
-  labels:
-    commcare.name: "elastic"
-  expose:
-    - "9200"
-
-elasticsearch_no_head:
-  image: elasticsearch:1.7.4
-  environment:
-    ES_JAVA_OPTS: "-Des.script.engine.groovy.inline.aggs=true -Des.script.engine.groovy.inline.search=true"
-  labels:
-    commcare.name: "elastic"
-  expose:
-    - "9200"
-
-web:
-  build: ../../
```

Comment:
This was problematic because it sets the root of this dockerfile at the repo root, which means the entire repo (minus /.git, which is in /.dockerignore) is sent to the docker daemon when this image is built. This makes the build take a long time, and as far as I can tell we don't need to send all those files to the docker daemon.